source/scss/_pf-basic-dropdowns.scss

```diff
@@ -0,0 +1,21 @@
+//
+// Dropdowns
+// --------------------------------------------------
+
+// Kebab dropdown
+.pf-dropdown__kebab {
+  .dropdown-toggle::after {
+    display: none;
+  }
+  .btn-link {
```

Comment:
related to this discussion: https://github.com/patternfly/patternfly-atomic/pull/25/files#r86324052

Instead of doing this:
.pf-dropdown__kebab .btn-link

I would use something like this?:
.pf-dropdown__kebab__btn-link

Is this the correct use of dashes and underscores?